woodwork/tests/accessor/test_table_accessor.py

```diff
@@ -2138,7 +2138,7 @@ def test_setitem_overwrite_column(sample_df):
     df.ww.init(use_standard_tags=False)
 
     original_col = df['full_name']
-    new_series = pd.Series([0, 1, 2], dtype='float')
+    new_series = pd.Series([0, 1, 2], dtype='float', name='full_name')
```

Comment:
Is it necessary to name the series here for the error message to match? If so, I suspect a common occurrence would be for unnamed Series to be assigned. Does the error make sense in that case?

Reply:
Without a name it probably doesn't make sense--`Standard tags have been added to None`. I'm thinking maybe we make the name parameter optional in in `StandardTagsChangedWarning` and just say `Standard tags have been added to your column` if there's no name.